packages/react-ui/__tests__/widgets/FormulaWidgetUI.test.js

```diff
@@ -36,6 +27,12 @@ describe('FormulaWidgetUI', () => {
     expect(await screen.findByText(DATA.value)).toBeInTheDocument();
   });
 
+  test('should render the current value', async () => {
+    const { rerender } = render(<FormulaWidgetUI data={1234} />);
+    rerender(<FormulaWidgetUI data={0} />);
```

Comment:
@AdriSolid I was thinking about specific tests, in a different file, for animations, not the test on the widget removing the mock, but I guess this is also nice (if we don't need to use a mock anymore)